phantomgames/Home/HomeScreenViewModel.swift

```diff
@@ -4,11 +4,10 @@
 //
 //  Created by Aphiwe Shozi on 2024/05/13.
 //
-
-import UIKit
-
-protocol HomeScreenViewModelDelegate: AnyObject {
+protocol HomeScreenViewModelDelegate: AnyObject { 
     func reloadView()
+    func reloadCollectionView()
+    func reloadTableView()
```

Comment:
You can remove reloadTableView since reloadView does reload the table view and these two are called together anyway

Reply:
agreed